packages/doenetml-worker-rust/lib-doenetml-core/src/core/general_prop/rendered_children_passthrough_prop.rs

```diff
@@ -37,25 +39,32 @@ impl RenderedChildrenPassthroughProp {
 
 /// Structure to hold data generated from the data queries
 #[derive(TryFromDataQueryResults, Debug)]
-#[data_query(query_trait = DataQueries, pass_data = &RenderedChildrenPassthroughProp)]
+#[data_query(query_trait = DataQueries, pass_data = &DataQuery)]
 struct RequiredData {
-    refs: PropView<prop_type::ContentRefs>,
+    refs: PropView<prop_type::AnnotatedContentRefs>,
 }
 impl DataQueries for RequiredData {
-    fn refs_query(arg: &RenderedChildrenPassthroughProp) -> DataQuery {
-        arg.data_query.clone()
+    fn refs_query(data_query: &DataQuery) -> DataQuery {
+        data_query.clone()
     }
 }
 
 impl PropUpdater for RenderedChildrenPassthroughProp {
-    type PropType = prop_type::ContentRefs;
+    type PropType = prop_type::AnnotatedContentRefs;
 
     fn data_queries(&self) -> Vec<DataQuery> {
-        vec![self.data_query.clone()]
+        RequiredData::data_queries_vec(&self.data_query)
     }
 
     fn calculate(&self, data: DataQueryResults) -> PropCalcResult<Self::PropType> {
         let required_data = RequiredData::try_from_data_query_results(data).unwrap();
-        PropCalcResult::Calculated(required_data.refs.value)
+        let children = required_data
```

Comment:
I'm trying to understand why you need all this rigamarole now and you didn't before. `RequiredData.refs.value` is an `Rc<AnnotatedContentRefs>`, which is the same as our `PropType`, i.e., `prop_type::AnnotatedContentRefs`. Why can't you just stick it in the `PropCalcResult::Calculated()`? If I revert this change, it still compiles and the test pass.

Reply:
I think that is left over from when I was converting `AnnotatedComponentRefs` back to `ComponentRefs`, just to make sure things were working. I will clean it up!